src/wallet/api/wallet2_api.h

```diff
@@ -161,6 +163,11 @@ struct UnsignedTransaction
     * return - true on success
     */
     virtual bool sign(const std::string &signedFileName) = 0;
+   /*!
+    * @brief sign - Sign txs and return as string
```

Comment:
Nitpick: `@brief sign` should match the function name like: `@brief signAsString()`